src/repos.json

```diff
@@ -54,9 +54,10 @@
     { "language": "ruby", "repo": "GoogleCloudPlatform/ruby-docs-samples"},
     { "language": "ruby", "repo": "GoogleCloudPlatform/getting-started-ruby"},
     { "language": "ruby", "repo": "googleapis/ruby-spanner-activerecord", "isTeamIssue": true},
-    
+
     { "language": "yaml", "repo": "GoogleCloudPlatform/workflows-samples"},
 
     { "language": "java", "repo": "GoogleCloudPlatform/cloud-bigtable-examples", "isTeamIssue": true}
+    { "language": "terraform", "repo": "GoogleCloudPlatform/anthos-samples", "isTeamIssue": true}
```

Comment:
- The repository mostly contains `yaml` and `terraform` scripts for now. It can include `go`/`python` scripts when tests are added.
- I wasn't sure what to include here; so added `terraform` for now hoping someone from the team can guide when reviewing the PR.